javaslang/src/main/java/javaslang/collection/AbstractMultimap.java

```diff
@@ -95,6 +95,12 @@ public ContainerType getContainerType() {
     }
 
     @Override
+    @SuppressWarnings("unchecked")
+    public Traversable<V> getOrElse(K key, Traversable<? extends V> defaultValue) {
```

Comment:
if you intend to return a `Traversable<V>`, shouldn't the default value be of that type also?